src/client/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -159,5 +159,11 @@ public partial interface IPublicClientApplication : IClientApplicationBase
             IEnumerable<string> scopes,
             string username,
             string password);
+
+        /// <summary>
+        /// Used to determine if the currently available broker is able to perform Proof-of-Possesion.
+        /// </summary>
+        /// <returns>Boolean indicating Proof-of-Possesion is supported</returns>
+        public bool IsProofOfPosessionSupportedByClient();
```

Comment:
Can it be `IsProofOfPossessionSupported` to keep it simple?

Reply:
This is technically not in the spec. Thoughts @bgavrilMS?